dashboard/test/ui/features/applab/htmlSanitization.feature

```diff
@@ -10,12 +10,19 @@ Feature: App Lab HTML Sanitization
     When I drag a SCREEN into the app
     And I drag a SCREEN into the app
     And I drag a LABEL into the app
+    # Set label text via the property editor instead of editing the DOM directly,
+    # if we ever figure out how to make React notice changes to input fields in IE.
+    And I set selector "#design_label1" text to ""
```

Comment:
I'm not entirely certain I'm parsing this comment correctly. Is it suggesting a potential better approach to this test in the future? If so, should we prepend it with something like "We should"... or "We could"...?